contracts/TokenManager.sol

```diff
@@ -10,29 +10,27 @@ contract TokenManager is Ownable {
 
     struct Token {
         address addr;
+        uint8 dec;
         address chainlinkAddr;
         uint8 chainlinkDec;
     }
 
     /// @param _wethAddress address of WETH token
+    /// @param _wethDec decimals of WETH token
     /// @param _ethUsdCL address of Chainlink data feed for ETH / USD
     /// @param _ethUsdCLDec number of decimals that ETH / USD data feed uses
-    constructor(address _wethAddress, address _ethUsdCL, uint8 _ethUsdCLDec) {
-        addDefaultTokens(_wethAddress, _ethUsdCL, _ethUsdCLDec);
+    constructor(address _wethAddress, uint8 _wethDec, address _ethUsdCL, uint8 _ethUsdCLDec) {
+        addDefaultTokens(_wethAddress, _wethDec, _ethUsdCL, _ethUsdCLDec);
     }
 
     // Gets the details for the given token, if it is accepted
     /// @param _name 32-byte array value representation of the token symbol e.g. "WETH", "USDT"
-    /// @return addr the address of the token
-    /// @return chainlinkAddr the address of the token / USD Chainlink data feed
-    /// @return chainlinkDec the number of decimals the Chainlink data feed uses
-    function get(bytes32 _name) external view returns(address addr, address chainlinkAddr, uint8 chainlinkDec) {
-        Token memory token = tokens[_name];
-        return (token.addr, token.chainlinkAddr, token.chainlinkDec);
+    function get(bytes32 _name) external view returns(Token memory) {
+        return tokens[_name];
     }
 
-    function addDefaultTokens(address _wethAddress, address _ethUsdCL, uint8 _ethUsdCLDec) private {
-        addAcceptedToken(bytes32("WETH"), _wethAddress, _ethUsdCL, _ethUsdCLDec);
+    function addDefaultTokens(address _wethAddress, uint8 _wethDec, address _ethUsdCL, uint8 _ethUsdCLDec) private {
```

Comment:
to clarify, you have already set the decimals for weth in the tests, `const WETH_DEC = 18` (as expected). if the weth token -- not the token it is wrapping -- always has 18 decimals, we won't need this additional `_wethDec` parameter. if we need a parameter due to varying decimals, we could rename it to something else.

Reply:
but this function is private, called through the constructor and passed to this function. that's where it's used in the test

Reply:
I'm just saying that in the constructor, you will "always" set the 2nd parameter to 18, if you want to adhere to weth decimals.
So `/// @param _wethDec decimals of WETH token` is redundant` since [WETH](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2) has 18 decimals.